cncf_process/incubation_proposal/incubation_proposal.md

```diff
@@ -0,0 +1,179 @@
+  - We enhanced the framework of the resource interpreter to enable users to customize the resource status collected to the control plane.
+  - We introduced the aggregated API to allow users to connect and operate member clusters from the control plane.
+  - We introduced a new component `karmada-search` to implement global search and global proxy for multi-cluster resources.
+  With these, users are freed from repetitive configurations and the management costs caused by fragmented API access portals.
+* **More convenient cross-cluster service governance**. We support multi-cluster service discovery and a one-stop multi-cluster network solution. Now users can enable service governance for cross-cluster with Karmada.
+* **High availability for multi-cloud applications**. We support cross-cluster failover of applications.
+  When a cluster fails, applications can smoothly migrate to an available cluster. During replicas migration, Karmada ensures that the service replicas do not drop to zero, thereby ensuring that the service will not be interrupted.
+  Based on this, users can use Karmada to manage distributed applications with confidence.
+* **Smoother multi-cluster O&M experience**. In the past year, We continue to optimize our command line tools. Now users can complete the creation of the Karmada control plane and bootstrap token-based cluster registration in just a few clicks.
+  Meanwhile, we provide not only a smooth version upgrade path but also a clear approach to migrate from Kubefed or promote legacy workloads from Kubernetes.
+
+## Incubation Stage Requirements
+
+**_Document that it is being used successfully in production by at least three independent end users which, in the TOC’s judgement, are of adequate quality and scope._**
+
+11 adopters are using Karmada in their `production` environments as of Oct 25, 2022. The full adopter list can be found on our [website](https://karmada.io/docs/next/casestudies/adopters).
+
+Highlighted examples include:
+
+* **[VIPKID](https://www.vipkid.com/)**: Uses Karmada to build a multi-cloud, multi-vendor PaaS platform across regions. The use case details can be found [here](https://karmada.io/docs/next/casestudies/vipkid).
+* **[ICBC](http://www.icbc.com.cn/ICBC/EN/)**: Uses Karmada to build a multi-cloud platform to manage **100** Kubernetes clusters and more than **280,000** containers. The use case details can be found [here](https://karmada.io/zh/blog/2022/03/22/multi-cluster-management-with-an-ocean-of-nodes/).
+* **[Vivo](https://www.vivo.com/)**: Builds a more resilient and elastic hybrid-multi-cloud platform using Karmada.
+* **[Trip.com](https://www.trip.com/)**: Uses Karmada to build a multi-cloud platform with cross-cluster auto scaling and scheduling.
+* **[AIML INSTITUTE](https://orang.cloud/)**: Uses Karmada to build multi-cluster platform with heterogeneous resources and clusters. The use case details can be found [here](https://karmada.io/docs/next/casestudies/ci123).
```

Comment:
Please sync the latest usage scenario from website.

Reply:
Updated.